libbeat/publisher/queue/diskqueue/consumer.go

```diff
@@ -86,10 +86,14 @@ func (batch *diskQueueBatch) Count() int {
 	return len(batch.frames)
 }
 
-func (batch *diskQueueBatch) Event(i int) interface{} {
+func (batch *diskQueueBatch) Entry(i int) interface{} {
 	return batch.frames[i].event
 }
 
+func (batch *diskQueueBatch) ID(i int) queue.EntryID {
+	return 0
```

Comment:
So the disk queue would not support the incremental IDs? Or it's going to be implemented in a separate PR?

Reply:
Yes, this is just for the memory queue, disk queue will be another PR (the shipper doesn't support the disk queue yet although with Lee's pending PR that should change any day now)